### PR TITLE
adds monthly aggregate delivery statement

### PR DESCRIPTION
**Pull request by mkhraisha**
Adds a new VC for Monthly Aggregate Delivery Statement

### CODE REVIEW

**Review by PatStLouis — APPROVED** [review]

File: docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+      enum:
+        - MonthlyAggregateDeliveryStatementCredential
+    deliveryStatements:
+      type: array
+      readOnly: true
+      $ref: ../common/DeliveryStatement.yml
+  proof:
+    $ref: ../snippets/proof.yml
+additionalProperties: false
+required:
```
Comment (nissimsan) [inline docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml:83]: @mkhraisha , id should be required! :D

File: docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  - '@context'
+  - type
+  - issuanceDate
+  - issuer
+  - credentialSubject
```
Comment (brentzundel) [inline docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml:88]:
```suggestion
required:
  - '@context'
  - type
  - id
  - issuanceDate
  - issuer
  - credentialSubject
```

File: docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  {
+    "type": [
+      "VerifiableCredential",
+      "MonthlyAggregateDeliveryStatementCredential"
+    ],
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/traceability/v1"
+    ],
+    "id": "did:web:organization.example.com",
```
Comment (PatStLouis) [inline docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml:100]:
```suggestion
    "id": "[suggest to change change for `https://organization.example.com/credentials/123`](https://organization.example.com/credentials/123)",
```

Comment (brentzundel) [reply docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml:100]: the other examples look like "id": "urn:credential:1a667a1c-f761-4172-b110-5610ad25f50b"

File: docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
```
Comment (brentzundel) [inline docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml:100]:
```suggestion
    "id": "urn:credential:1a667a1c-f761-4172-b110-5610ad25f50b",
```

**Review by nissimsan — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

File: docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+    default:
+      - MonthlyAggregateDeliveryStatementCredential
+    items:
+      type: string
+      enum:
+        - MonthlyAggregateDeliveryStatementCredential
+    deliveryStatements:
+      type: array
+      readOnly: true
+      $ref: ../common/DeliveryStatement.yml
```
Comment (PatStLouis) [inline docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml:79]:
```suggestion
    type:
      type: array
      readOnly: true
      const:
        - MonthlyAggregateDeliveryStatementCredential
      default:
        - MonthlyAggregateDeliveryStatementCredential
      items:
        type: string
        enum:
          - MonthlyAggregateDeliveryStatementCredential
      deliveryStatements:
        type: array
        readOnly: true
        $ref: ../common/DeliveryStatement.yml
```

**Comment by nissimsan** [comment]
@mkhraisha, there were some indenting issues on this. We tried to fix it live on the meeting, but it's still broken. Will you take a look, pls?

File: docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml (hunk 1 above)
Comment (mkhraisha) [reply docs/openapi/components/schemas/credentials/MonthlyAggregateDeliveryStatements.yml:82]: Done!

**Comment by mkhraisha** [comment]
Fixes linting issues, adds required ID

**Pull request merged**